include/ensmallen_bits/adam/adam.hpp

```diff
@@ -63,7 +64,7 @@ namespace ens {
  *
  * @tparam UpdateRule Adam optimizer update rule to be used.
  */
-template<typename UpdateRule = AdamUpdate>
+template<typename UpdateRule = AdamUpdate,typename DecayPolicyType = NoDecay>
```

Comment:
Missing space after, ``,``, also can you add a comment for the ``DecayPolicyType`` template parameter.